core/cron/nests.cron.php

```diff
@@ -17,8 +17,17 @@
 	$nests['lng'] = $data->longitude;
 	$starttime = $data->latest_seen - $data->duration;
 
-	$nests['st'] = date("i",$starttime);;
-	$nests['et'] = date("i",$data->latest_seen);;
+	$startdate = date("i",$starttime);
+	$enddate = date("i",$data->latest_seen);
+
+	if ($enddate < $startdate) { // Smaller number first
```

Comment:
I don't think this is correct, since we can have spawns from :50 to :20

Reply:
oh yeah right.
Woops